scripts/components/advanced-color-picker/advanced-color-picker.js

```diff
@@ -138,7 +140,7 @@ export const AdvancedColorPicker = (props) => {
 			}
 
 			{type === 'gradient' && showGradient && !disabled &&
-				<GradientPicker
+				<GradientPickerComponent
```

Comment:
what are the changes implemented in the newer version so we need to reference 2 different gradient component?
Are the only names changed or?

Reply:
ok